ksqldb-rest-app/src/main/java/io/confluent/ksql/api/auth/KsqlAuthorizationProviderHandler.java

```diff
@@ -32,7 +32,7 @@
  */
 public class KsqlAuthorizationProviderHandler implements Handler<RoutingContext> {
 
-  public static final Set<String> PATHS_WITHOUT_AUTHORIZATION = ImmutableSet
+  public static final Set<String> KSQL_AUTHENTICATION_SKIP_PATHS = ImmutableSet
```

Comment:
Can we move this into `AuthenticationPluginHandler` if we're going to rename it? It looks out of place here.

Reply:
done